### PR TITLE
Removed Items from authorization creatable.

### PR DESCRIPTION
**Pull request by marcushanses**

### CODE REVIEW

File: Authorization/Creatable/Base.ts [hunk 1]
```diff
@@ -27,7 +27,8 @@ export namespace Base {
 			((value.amount != undefined && value.items == undefined) ||
 				(value.amount == undefined && value.items != undefined)) &&
 			(value.amount == undefined || (typeof value.amount == "number" && !isNaN(value.amount))) &&
-			(value.items == undefined || base.Item.canBe(value.items)) &&
+			(value.items == undefined || (typeof value.items == "number" && !isNaN(value.items))) &&
+			!value.amount != !value.items &&
```
Comment (nilssonemma) [inline Authorization/Creatable/Base.ts:31]: This logic is already written above

File: Authorization/index.ts [hunk 2]
```diff
@@ -21,7 +21,6 @@ export interface Authorization {
 	reference: string | { network: string; processor: string }
 	created: isoly.DateTime
 	amount: number
```
Comment (nilssonemma) [inline Authorization/index.ts:23]: Please update is and flaw function in this file

File: State/Authorization.ts [hunk 3]
```diff
@@ -106,8 +106,7 @@ export namespace Authorization {
 				number: state.authorization.number,
 				reference: state.authorization.reference,
 				created: state.authorization.created,
-				amount: state.authorization.amount,
-				items: state.authorization.items,
+				amount: state.authorization.amount ?? Items.from(state.authorization.items),
```
Comment (nilssonemma) [inline State/Authorization.ts:109]: Is there a reason items is not removed from the state?

**Review by nilssonemma — CHANGES_REQUESTED** [review]

File: Authorization/Creatable/Base.ts (hunk 1 above)
Comment (nilssonemma) [reply Authorization/Creatable/Base.ts:31]: By "this logic" I mean where you check that only one of items/amount is assigned

**Comment by nilssonemma** [comment]
Please also update to latest prettierx/eslint to fix audit error

File: Authorization/Creatable/Base.ts [hunk 4]
```diff
@@ -24,10 +24,10 @@ export namespace Base {
 			value &&
 			(value.id == undefined || authly.Identifier.is(value.id, 16)) &&
 			typeof value.number == "string" &&
-			((value.amount != undefined && value.items == undefined) ||
-				(value.amount == undefined && value.items != undefined)) &&
+			!value.amount != !value.items &&
 			(value.amount == undefined || (typeof value.amount == "number" && !isNaN(value.amount))) &&
-			(value.items == undefined || base.Item.canBe(value.items)) &&
+			(value.items == undefined || (typeof value.items == "number" && !isNaN(value.items))) &&
+			!value.amount != !value.items &&
```
Comment (nilssonemma) [inline Authorization/Creatable/Base.ts:30]:
```suggestion
```

**Review by nilssonemma — CHANGES_REQUESTED** [review]

File: Authorization/Creatable/Base.ts (hunk 4 above)
Comment (marcushanses) [reply Authorization/Creatable/Base.ts]: I realized this check will not work for 0 amounts.

**Review by nilssonemma — APPROVED** [review]

**Pull request merged**